ZenPacks/zenoss/OpenStack/tests/test_modeler.py

```diff
@@ -0,0 +1,127 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import os
+import logging
+import json
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger('zen.OpenStack')
+
+from zope.event import notify
+
+import Globals
+
+from Products.Zuul.catalog.events import IndexingEvent
+from Products.Five import zcml
+
+from Products.DataCollector.ApplyDataMap import ApplyDataMap
+from Products.ZenTestCase.BaseTestCase import BaseTestCase
+from Products.ZenUtils.Utils import unused
+
+import ZenPacks.zenoss.OpenStack
+from ZenPacks.zenoss.OpenStack.tests.utils import setup_crochet
+from ZenPacks.zenoss.OpenStack.modeler.plugins.zenoss.OpenStack \
```

Comment:
it must look like this:
from ZenPacks.zenoss.OpenStack.modeler.plugins.zenoss.OpenStack \
    import OpenStack as OpenStackModeler